container_files/public_html/doc/builtin/procedures/SplitProcedure.md

```diff
@@ -0,0 +1,18 @@
+# Dataset Split Procedure
+
+This procedure splits a dataset into several while respecting the distribution 
+of the presence of one or several labels.
+
+This can be used, for example, to split a dataset with multiple labels
+between training, testing and validation sets.
+
+## Algorithm
+
+The procedure uses a greedy algorithm and is not guaranteed to return the optimal solution.
+The algorithm will try to assure than every label is represented in all splits.
+If a label is represented in fewer rows than there are splits, the rows will be placed in the 
+firsts splits.
```

Comment:
Seems like it would always throw in unbalanced sets. Maybe an option to skip them? I'll need two passes.